tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3120,57 +3124,31 @@ public function userTriesToSendGroupCreationRequestUsingTheAPI(string $user, str
 	 */
 	public function createTheGroup(string $group, ?string $method = null):void {
 		//guess yourself
-		if ($method === null) {
-			if ($this->isTestingWithLdap()) {
-				$method = "ldap";
-			} elseif (OcisHelper::isTestingWithGraphApi()) {
-				$method = "graph";
-			} else {
-				$method = "api";
-			}
-		}
 		$group = \trim($group);
-		$method = \trim(\strtolower($method));
 		$groupCanBeDeleted = false;
 		$groupId = null;
-		switch ($method) {
```

Comment:
keep the switch statement and only remove the `api` case

Reply:
```suggestion
		$fullUrl = $this->getBaseUrl() . "/graph/v1.0/groups/$group";
```
@saw-jan can we do this